r/tests/testthat/test-gcs.R

```diff
@@ -17,6 +17,8 @@
 
 skip_if_not_available("gcs")
 
+source_file("helper-filesystems.R")
```

Comment:
Aren't all helper files loaded by testthat anyway? (i.e. you shouldn't need this)

Reply:
For some reason I found I had to add it. Let me try removing it again and see what happens.

Reply:
You are correct